src/test/java/emissary/test/core/junit5/UnitTest.java

```diff
@@ -52,6 +52,7 @@ public abstract class UnitTest {
 
     @TempDir
     public static File temporaryDirectory;
+    @SuppressWarnings("ConstantField")
     protected static String TMPDIR = "/tmp";
```

Comment:
Any reason why this field can't renamed to comply with the naming convention of a non-constant field.
```suggestion
    protected static String tmpDir = "/tmp";
```

Reply:
> Any reason why this field can't renamed to comply with the naming convention of a non-constant field.

Just potential integration cost

Reply:
Correct, it just has enough touch points downstream that it spilled over the acceptable arbitrary limit.